pkg/plugins/common.go

```diff
@@ -36,7 +37,9 @@ func templateSysData(l logger.Interface, s string) string {
 		l.Warn(fmt.Sprintf("Failed marshalling '%s': %s", s, err.Error()))
 		return s
 	}
-	l.Debug(string(data))
+	litter.Config.HideZeroValues = true
+	litter.Config.HidePrivateFields = true
+	l.Trace(litter.Sdump(&system))
```

Comment:
same as the other but put this on trace level. We dont usually need this as part of the debug part and its very ugly,big and pollutes the log with a lot of useless, repeated info.